### DISCUSSION TITLE
Long Range DeepMD training data

### DISCUSSION THREAD

**Discussion by mayankaditya** [deepmodeling/deepmd-kit#1533] [General]
Dear All,

I am interested in the simulation of polar materials. I would like to know how to generate the data (AIMD data) for DPLR training and perform the training. An example of the same would be very helpful.

Thanks,
Mayank

**Comment by njzjz** [comment]
See #1458.

**Comment by mayankaditya** [comment]
Hi Jinzhe Zeng,

Thanks for the information. I am wondering how to extract the polarizability/dipole data from AIMD. Is it implemented in dpdata module?

Thanks,
Mayank

**Reply by njzjz** [reply]
> Is it implemented in dpdata module?

I don't think so.

>  I am wondering how to extract the polarizability/dipole data from AIMD.

Maybe @amcadmus can answer the question..

**Reply by wanghan-iapcm** [reply]
One computes the Wannier centroid (WC) by tools like wannier90, and defines the "dipole" as the displacement of the WC with respect to the atom to which it is associated. This is explained in https://arxiv.org/pdf/2112.13327.pdf section II
